### DISCUSSION TITLE
macOS: Open disambiguation selection on active monitor

### DISCUSSION THREAD

**Discussion by EmSixTeen** [espanso/espanso#1462] [Feature requests and Ideas]
Hey there! Not sure if it's intended, if I've maybe missed an option or something, but when I expand a basic shortcut which causes the Match Disambiguation selection window to pop up it always pops up on the main screen rather than the currently active one where my text input actually is. Would be nice if that were the case. 😊

OS: macOS Monterey 12.1 (M1)
Espanso: 2.1.8 

Sidenote: Espanso version was annoying to find, ended up finding it by right clicking the app in Applications and finding it listed under Get Info.

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

Same here on Linux. I hadn't thought about it before, as it works in my setup, but ideally it would be configurable.

The `espanso` command currently shows the version but an `espanso version` command would be ideal.
